cypress/e2e/read-receipts/high-level.spec.ts

```diff
@@ -2344,12 +2369,236 @@ describe("Read receipts", () => {
         it.skip("A receipt for the last unimportant event makes the room read, even if all are unimportant", () => {});
     });
 
+    function assertMessageLoaded(messagePart: string) {
+        cy.get(".mx_EventTile_body").contains(messagePart).should("exist");
+    }
+
+    function assertMessageNotLoaded(messagePart: string) {
+        cy.get(".mx_EventTile_body").contains(messagePart).should("not.exist");
+    }
+
+    function pageUp() {
+        cy.get(".mx_RoomView_messagePanel").then((refs) =>
+            refs.each((_, messagePanel) => {
+                console.log(`AJB ${messagePanel.scrollTop}`);
```

Comment:
Should these still be here? Eventually convert hem to common log messages.

Reply:
Thank you, accidentally left in. Now deleted.